extras/util/permission.js

```diff
@@ -0,0 +1,36 @@
+const headTrainer = 'head-trainer';
+const trainer = 'trainer';
+const trainee = 'trainee';
+
+let permissions = {
+  'getUsers1': {
+    all: [headTrainer],
+    read: [trainee, trainer],
+    write: [trainer],
+    delete: [],
+  }, 'getUsers2': {
+    all: [trainer],
+    read: [trainee],
+    write: [trainer],
+    delete: [],
+  }, 'getUsers3': {
+    all: [headTrainer],
+    read: [trainee, trainer],
+    write: [trainee],
+    delete: [],
+  }
+}
+function hasPermissions( moduleName, role, permissionType ) {
+  if(permissions[moduleName]) {
```

Comment:
@pvvarun You can reduce the if else block here. Just return the condition which you are using in if block.